### PR TITLE
Document how to change MySQL passwords

### PR DESCRIPTION
**Pull request by rhcarvalho**
Part of https://trello.com/c/CsdmW7nJ

@bparees @mfojtik @adellape PTAL

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@mnagy FYI

**Comment by bparees** [comment]
sentence about unsetting ROOT password?

**Comment by rhcarvalho** [comment]
@bparees https://github.com/openshift/openshift-docs/pull/939/files#diff-24bb9b5c5fb226bdba10353136a95468R114

**Comment by bparees** [comment]
@rhcarvalho i know, but i think it would be useful to also note it when discussing password changing.

File: using_images/db_images/mysql.adoc [hunk 1]
```diff
@@ -160,6 +160,83 @@ the database:
...
+changing the environment variables `*MYSQL_PASSWORD*` and
+`*MYSQL_ROOT_PASSWORD*`, respectively.
+
+Whenever `*MYSQL_ROOT_PASSWORD*` is set, it enables remote access for the root
+user with the given password, and whenever it is unset, remote access for the
+root user is disabled.
+This doesn't affect the regular user `*MYSQL_USER*` who always has remote
+access.
+This also does not affect local access by the root user, who can always login
+without password in *localhost*.
```
Comment (rhcarvalho) [inline using_images/db_images/mysql.adoc]: @bparees how about the paragraph above?

Comment (bparees) [inline using_images/db_images/mysql.adoc]: awesome.

**Comment by bparees** [comment]
lgtm.

**Comment by rhcarvalho** [comment]
@adellape PTAL

**Comment by rhcarvalho** [comment]
@adellape @CowboysFan PTAL

File: using_images/db_images/mysql.adoc [hunk 2]
```diff
@@ -160,6 +160,83 @@ the database:
...
+
+You can verify that the new password is in effect with:
+
+----
+$ oc get pods
+...
+$ oc exec <DB_PODNAME> -- bash -c 'mysql -u $MYSQL_USER -pNewPassword -h mysql $MYSQL_DATABASE -te "SELECT * FROM (SELECT database()) db CROSS JOIN (SELECT user()) u"'
+----
+
+Replace *<DB_PODNAME>* with the name the running MySQL pod.
```
Comment (tnguyen-rh) [inline using_images/db_images/mysql.adoc]: "with the name of" (missing preposition)

**Comment by rhcarvalho** [comment]
@vikram-redhat PTAL

File: using_images/db_images/mysql.adoc (hunk 2 above)
Comment (rhcarvalho) [inline using_images/db_images/mysql.adoc]: Excellent catch, thanks! Fixed also the missing prepositions on similar lines for MongoDB and PostgreSQL.

**Comment by rhcarvalho** [comment]
@nhr do you know who can verify/merge this?

**Comment by nhr** [comment]
@rhcarvalho - it follows the merge process for the openshift-server repo, to wit: [merge]

**Comment by mfojtik** [comment]
@nhr thought content guys merge here :-)

**Comment by nhr** [comment]
@mfojtik This is strictly Origin M4 documentation, but @adellape feel free to throw things at me if I jumped the gun.

**Comment by rhcarvalho** [comment]
@nhr there's no bot here AFAIK, so [merge] doesn't work :smile:

**Comment by mfojtik** [comment]
maybe my [merge] is more powerful ;-)

**Comment by nhr** [comment]
I laugh at your inferior [merge] powers :-) @adellape - I have merge rights on this so please give me a thumbs-up if you are okay with the changes.

**Comment by mfojtik** [comment]
@nhr my merge-fu gets weaker :/

**Comment by nhr** [comment]
@mfojtik You've transcended. Minions do the merging for you now :-)

**Comment by bparees** [comment]
technically i have merge rights but i prefer to let the content guys own merging on this repo.

and no, there's no merge bot that i'm aware of for this one.

**Comment by vikram-redhat** [comment]
Sorry guys. Alex is on PTO. I will view and merge later today.

**Pull request merged**